micro-ui/web/micro-ui-internals/packages/modules/core/src/config/LandingPageConfig.js

```
@@ -0,0 +1,50 @@
+export  let config = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'config' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)